src/megaapi_impl.cpp

```diff
@@ -14276,14 +14379,33 @@ void MegaApiImpl::getua_result(TLVstore *tlv, attr_t)
                 }
                 break;
             }
+
+            case MegaApi::USER_ATTR_ALIAS:
+            {
+                // If a uh was set in the request we have to find it in the aliases map and return it
+                const char *uh = request->getText();
+                if (uh)
+                {
+                    request->setMegaStringMap(NULL);
```

Comment:
I think we can keep the original object in there, it could be useful, despite we don't expose it in the documentation, but internally. I'll change it ;)